README.md

```diff
@@ -13,26 +13,33 @@ The Identity Platform Login UI Operator may be deployed using with the following
 juju deploy identity-platform-login-ui-operator
 ```
 
-## Interacting with Kratos and Hydra
+## Relations
 
-To set the Ory Kratos endpoint to the UI server, do:
+### Hydra
+
+This charm offers integration with [hydra-operator](https://github.com/canonical/hydra-operator). To integrate them, run:
 
 ```console
-juju config identity-platform-login-ui-operator kratos_url={kratos_url:port}
+juju integrate kratos:kratos-endpoint-info identity-platform-login-ui-operator:kratos-endpoint-info
+juju integrate kratos:ui-endpoint-info identity-platform-login-ui-operator:ui-endpoint-info
```

Comment:
This section talks about hydra, but the commands are for kratos

Reply:
fixed, sorry about that